dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -0,0 +1,53 @@
+class EvaluateRubricJob < ApplicationJob
+  queue_as :default
+
+  # To make this job get run in development, you have two options:
+  # 1. switch the queue_adapter value here to :async, or
+  # 2. leave the value as :delayed_job, and run the delayed job worker locally
+  #    via `dashboard/bin/delayed_job restart` or rake build
+  self.queue_adapter = :delayed_job
```

Comment:
@wilkie @bethanyaconnor seems like staging will be closed for a while so please let me know if you have any opinions here. goals are:
1. run this job via delayed job in production and other environments (at least staging, test, adhoc)
2. provide some way to run this job locally

currently we're starting the delayed job worker in every environment (including development) during `rake build`: https://github.com/code-dot-org/code-dot-org/blob/2de65f17160c9288170d962a55ac615607f63e4c/lib/rake/build.rake#L113-L114

If this feels like it could turn into a bigger change I'm happy to back these changes out of this PR or make an adjustment in a follow-up PR.